pkg/integrations/prometheus/promql/metricsSearchHandler.go

```diff
@@ -216,6 +290,93 @@ func ProcessMetricsSearchRequest(ctx *fasthttp.RequestCtx, myid uint64) {
 	WriteJsonResponse(ctx, &mQResponse)
 	ctx.SetContentType(ContentJson)
 	ctx.SetStatusCode(fasthttp.StatusOK)
+
+}
+func ProcessPromqlBuildInfoRequest(ctx *fasthttp.RequestCtx, myid uint64) {
+	buildInfo := map[string]interface{}{
+		"status": "success",
+		"data": map[string]string{
+			"version":   "2.23.1",
+			"revision":  "cb7cbad5f9a2823a622aaa668833ca04f50a0ea7",
+			"branch":    "master",
+			"buildUser": "julius@desktop",
+			"buildDate": time.Now().Format("20060102-15:04:05"),
+			"goVersion": runtime.Version(),
```

Comment:
hardcode the go version,

Reply:
define this `PromQLBuildInfoStr1 as a const string at the top of the file and use it directly at line number 315.
However write a quick UT, to parse this json const (just like what we are doing for promql functions json str